plugins/broker/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
 cmake_minimum_required(VERSION 3.19...3.24 FATAL_ERROR)
+message(
+  WARNING "broker plugin disabled due to lack of compatibility with CAF 18.6")
+return()
```

Comment:
Instead of this I propose that we remove the code outright and create a story for bringing it back. This also needs its own changelog entry. I propose the following:

> We removed the `broker` plugin that enabled direct Zeek 3.x log transfer to VAST. The plugin will return in the future written for Zeek 5+.

Reply:
I will make a separate PR for this change

Reply:
Before i merge this one i mean